homeassistant/components/climate/strings.json

```diff
@@ -161,19 +161,19 @@
     },
     "set_temperature": {
       "name": "Set target temperature",
-      "description": "Sets target temperature.",
+      "description": "Sets the temperature setpoint.",
       "fields": {
         "temperature": {
-          "name": "Temperature",
-          "description": "Target temperature."
+          "name": "Target temperature",
+          "description": "The temperature setpoint."
         },
         "target_temp_high": {
-          "name": "Target temperature high",
-          "description": "High target temperature."
+          "name": "Upper target temperature",
+          "description": "The highest temperature allowed."
```

Comment:
I'm not sure if `allowed` terminology is right here. It is a setpoint.
The `max` still applies (and is different).